app/utils.py

```diff
@@ -24,3 +24,18 @@ def assess_contact_type(service_contact_info):
         return "link"
     else:
         return "other"
+
+
+def bytes_to_pretty_file_size(bytes):
+    if bytes < 1024 / 20:
+        # File less than 0.05KB (one twentieth of a KB) to be represented in B
+        # Anything bigger will round at 1dp to at least 0.1KB
+        return str(bytes) + "B"
+    if bytes < (1024**2) / 20:
+        # File less than 0.05MB to be represented in KB
+        # Anything bigger will round at 1dp to at least 0.1MB
+        kb_to_1dp = round(bytes / 1024, 1)
+        return str(kb_to_1dp).rstrip(".0") + "KB"
+    else:
```

Comment:
Absolutely trivial and optional nit, but this reads kind of weird because we have one isolated if statement at the beginning and then an if/else following it, which feels a little inconsistent. Might be nicer as an if/elif/else, or just an if/if/(no else) return.

Always think it's nice to have a catch-all return at the end of a function not nested; same effect but is slightly clearer that there's no edge case that could implicitly return None.

Reply:
I like this suggestion. I'm going to go for the if/elif/else as the default can be to always fall back to MB